laghos.cpp

```diff
@@ -605,7 +613,27 @@ int main(int argc, char *argv[])
    BlockVector S_old(S);
    long mem=0, mmax=0, msum=0;
    int checks = 0;
-
+   //   const double internal_energy = hydro.InternalEnergy(e_gf);
```

Comment:
If this commented block of code should be kept for convenience, it would be better to use a `bool` to decide whether to do the output rather than commenting out.

Reply:
These changes made to the main directory are actually merged from the `master` branch. I guess we should better not make any changes so that when we merge from the `master` in the future, it would not cause any conflicts.

Reply:
Yes, I see now that it is in master, which is surprising. I agree to keep it, to avoid conflicts.